src/bindeps_integration.jl

```diff
@@ -35,7 +35,7 @@ end
 
 libdir(p::HB, dep) = joinpath(brew_prefix, "lib")
 
-provider(::Type{HB}, packages::Vector{ASCIIString}; opts...) = HB(packages)
+provider(::Type{HB}, packages::Vector{String}; opts...) = HB(packages)
```

Comment:
Use a type parameter. This is guaranteed to be breaking. If this function is used at all.